maven-3.5-jdk8-alpine/README.md

```diff
@@ -0,0 +1,55 @@
+[mavan](https://maven.apache.org/) docker image, able to manage a project's build.
```

Comment:
- Typo on maven
- the fact that the image manages custom UID/GID should be mentioned, since this is the reason why we re-packaged the image in the 1<sup>st</sup> place

Reply:
- Typo fixed
- Add a sentence mentionning the abitlity of the image of managing UID/GID